src/blocks/accordion-item/index.js

```diff
@@ -26,14 +26,14 @@ registerBlockType( metadata.name, {
 		const blockProps = useBlockProps.save( {
 			className: 'wmf-accordion-item',
 		} );
-		const { fontColor, title } = attributes;
+		const { borderColor, title } = attributes;
 
 		return (
-			<div { ...blockProps }>
-				<button
-					className="wmf-accordion-item__title"
-					style={ fontColor && { color: fontColor } }
-				>
+			<div
+				{ ...blockProps }
+				style={ borderColor && { borderLeftColor: borderColor } }
+			>
+				<button className="wmf-accordion-item__title">
```

Comment:
Does this cause a deprecation warning when opening the prior year's report?

Reply:
Yes, though the content here will need updating to apply the border colour for the old report anyway. Feels like a block recovery would be okay in these instances, since we only have one page to maintain?

Reply:
@sambulance I think it's always a better UX to handle the deprecation, block recovery if the user isn't expecting it looks like a bug and is avoidable when the block can be upgraded automatically on save.

Reply:
Added, thank you both